src/ui/components/Popup/Popup.jsx

```diff
@@ -69,11 +69,24 @@ const popupInner = css({
   },
 });
 
+const popupInnerLevelEnd = css({
+  position: 'relative',
+  width: 'clamp(500px, 60vw, 660px)',
+  border: `${POPUP_INNER_BORDER_WIDTH} solid`,
+  borderImageSlice: 1,
+  borderImageSource: theme.colors.gradient.steel,
+  backgroundColor: theme.colors.primary.main,
+  margin: '5rem 0',
+  '@media (max-width: 1200px)': {
+    width: '94%',
+  },
```

Comment:
Z tym bym coś zrobił, bo to wygląda jakby miało prawo się popsuć przy minimalnej zmianie layoutu.